src/usr/local/containerbase/tools/v2/conan.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# shellcheck source=/dev/null
+. "$(get_containerbase_path)/utils/python.sh"
+
+function link_tool () {
+  post_install
+  [[ -n $SKIP_VERSION ]] || conan --version
+  conan profile detect
```

Comment:
why this?

Reply:
To properly test Conan, a Conan profile is required. However, the Conan profile depends on the processor architecture and compiler version, and the compiler version varies among Ubuntu releases. So IMHO `conan profile detect` is the easiest and most elegant way to get a Conan profile.